scania-digital-design-system/tegel
# test(button): added unit tests

**Describe pull-request**  
Followed the test steps in ticket/excel to test.

**Solving issue**  
Fixes: [CDEP-2961](https://tegel.atlassian.net/browse/CDEP-2961)

**How to test**  
1. Navigate to packages/core.
2. Check the code for tests in button component.
3.  Make sure docker is running.
4. Then run `npm run test` in core to run the tests. Make sure they run.

**Additional context** 
I'm not sure how we would like to check for colors on the button, but this way I found works for me.
I'm 100% sure I missed some tests and that some tests dont make sense. But they all run and will break if something changes on the button that is changing.
We are still in exploration for good testing.


[CDEP-2961]: https://tegel.atlassian.net/browse/CDEP-2961?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
